views/forms/ChartForm.jsx

```diff
@@ -12,6 +12,7 @@ import { chartStyles } from '../styles/chart_styles'
 const ChartForm = ({ classes, handleSubmit }) => {
   const [formValues, setFormValues] = useState({
     title: '',
+    description: '',
     assessment: '',
     variable: '',
     fieldLabelValueMap: [
```

Comment:
Not a blocker. If you've got `useState` object values, you may run into issues with `useEffect` hooks that depend on those object values. For example, `useEffect` with `formValues` in its dependency array may not reevaluate when `formValues.title` changes. Run into this occasionally, so just wanted to point it out; something to keep in the back of your mind if this is a common pattern throughout the app.